docker/compose/monitoring/docker-compose.mon.yaml

```diff
@@ -0,0 +1,82 @@
+# To start the services, run
```

Comment:
why .mon.yaml? it is already namespaced via the folder.

Reply:
so that it doesn't have same name as other docker compose file, then make it easy for people to just symlink it into the folder where they want to use it

Reply:
symbolic links can have different names then the source file, this feels non-uniform with the rest of the docker composes and I love consistency